### ISSUE TITLE
Scale exported game

### ISSUE THREAD

**Issue by toolness** [toolness/mobile-microgame-maker#6]
It would be nice if the exported game scaled proportionally to the size of the window it was in.

Phaser's [ScaleManager](http://docs.phaser.io/Phaser.ScaleManager.html) might help with this but it seems confusing to use. Even the [example](http://examples.phaser.io/_site/view_full.html?d=input&f=game+scale.js&t=game%20scale) doesn't seem to actually work until you start manually resizing the window.

**Comment by toolness** [comment]
It seems Phaser 2.2.1, released on December 4, actually vastly improves the ScaleManager, so hopefully this should be easier now.

**Comment by toolness** [comment]
It looks like putting the game in a parent element that has `height: 100%; width: 100%; overflow: hidden` and adding the following to the `preload` does the trick:

``` javascript
  game.scale.pageAlignHorizontally = true;
  game.scale.pageAlignVertically = true;
  game.scale.maxWidth = 800;
  game.scale.maxHeight = 600;
  game.scale.scaleMode = Phaser.ScaleManager.SHOW_ALL;
  game.scale.setScreenSize();
```

**Comment by toolness** [comment]
Oops, I guess this got fixed by #22.
